s3upload.js

```diff
@@ -60,7 +60,7 @@ S3Upload.prototype.createCORSRequest = function(method, url) {
 S3Upload.prototype.executeOnSignedUrl = function(file, callback) {
     var xhr = new XMLHttpRequest();
     var fileName = file.name.replace(/\s+/g, "_");
-    xhr.open('GET', this.signingUrl + '?objectName=' + fileName, true);
+    xhr.open('GET', this.signingUrl + '?objectName=' + fileName + '&contentType=' + file.type, true);
```

Comment:
If file.type is not supported then we will have issues either way.  Is this something we want to try to support?

We can leave as-is or maybe use the mime library on both sides.